cmd/fluxctl/root_cmd.go

```diff
@@ -70,6 +71,8 @@ func (opts *rootOpts) Command() *cobra.Command {
 		PersistentPreRunE: opts.PersistentPreRunE,
 	}
 
+	cmd.PersistentFlags().StringVar(&opts.Context, "context", "",
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&opts.Context, "k8s-context", "",
```

Reply:
to follow kubectl semantics i suggest not adding `k8s-`

Reply:
I think it's more important to make the flag consistent with the rest of the kubernetes flags (which are `k8s`-prefixed)

Reply:
Uhm, although we already have `--kube-config`. @stefanprodan / @hiddeco what do you think?

Reply:
I would keep the `k8s` prefix only for the port forward component, we used that prefix to avoid collision with the existing `namespace` flag but I see no reason to use that prefix for context.

Reply:
I am on the fence because `--context` is very generic.

`--k8s-context` makes more sense from my perspective as an engineer and maintainer, as the flag controls not _Flux' context_ but the _Kubernetes context_ that should be used to perform an action with `fluxctl`, and while `--kube-config` may not be prefixed with `k8s`, the `kube` still classifies what it controls (and that it is not directly related to Flux).

From a user's perspective however, it would make more sense to just go for `--context` so that I would not have to think or get frustrated.

Reply:
Helm offers the same functionality and uses `--kube-context`. 

```console
hidde@baldr:~$ helm get -h

...

Flags:
  -h, --help   help for get

...
      --kube-context string              name of the kubeconfig context to use
      --kubeconfig string                path to the kubeconfig file

```

This would be in line with our `--kube-config` flag.